tox.ini

```diff
@@ -36,22 +33,61 @@ changedir = .tmp/{envname}
 #
 description =
     run tests
+    devdeps: with the latest developer version of key dependencies
+    oldestdeps: with the oldest supported version of key dependencies
+setenv =
+    PYTHONPATH = sunkit-instruments/sunkit_instruments
+    MPLBACKEND = agg
+    COLUMNS = 180
+    PYTEST_COMMAND = pytest test.py
+    astropydev,build_docs,online: HOME = {envtmpdir}
 
+# install testing framework
+# ... or install anything else you might need here
 deps =
+    # All our tests run in parallel which this plugin allows us to.
+    pytest-xdist
+    # The devdeps factor is intended to be used to install the latest developer version.
+    # of key dependencies.
+    devdeps: cython
+    devdeps: git+https://github.com/astropy/astropy
+    devdeps: git+https://github.com/sunpy/ndcube
+    devdeps: git+https://github.com/sunpy/sunpy
+    ndcube: git+https://github.com/sunpy/ndcube@1.3
```

Comment:
We don't need to test against the ndcube 1.3 branch in this package.  The stable version and master will suffice.